examples/motor-imagery/plot_frequency_band_selection.py

```diff
@@ -0,0 +1,223 @@
+                             return_class_dis=True, verbose=False)
+
+print('Selected frequency band : ' + str(best_freq[0])
+      + '-' + str(best_freq[1]) + ' Hz')
+
+###############################################################################
+# Step2: Train classifier using selected frequency band and evaluate
+# performance using test set
+
+# Apply band-pass filter using the best frequency band
+best_raw_filter = raw.copy().filter(best_freq[0], best_freq[1],
+                                    method='iir', picks=picks,
+                                    verbose=False)
+
+events, _ = events_from_annotations(best_raw_filter, event_id,
+                                    verbose=False)
+
+# Read epochs (train will be done only between 0.5 and 2.5s)
+epochs = Epochs(
+    best_raw_filter,
+    events,
+    event_id,
+    tmin,
+    tmax,
+    proj=True,
+    picks=picks,
+    baseline=None,
+    preload=True,
+    verbose=False)
+
+# Get epochs
+epochs_data_train = 1e6 * epochs.get_data()
+
+# Estimate covariance matrices
+cov_data = Covariances().transform(epochs_data_train)
+
+# Classification with minimum distance to mean
+model.fit(cov_data[train_ind], labels[train_ind])
+acc = model.score(cov_data[test_ind], labels[test_ind])
+
+t3 = time() - t2
+
+###############################################################################
+# Compare pipelines: accuracies and training times
+# ------------------------------------------------
+
+print("Classification accuracy without frequency band selection: "
+      + f"{acc_baseline:.02f}")
+print("Total computational time without frequency band selection: "
+      + f"{t1:.5f} s")
+print("Classification accuracy with frequency band selection: "
+      + f"{acc:.02f}")
+print("Total computational time with frequency band selection: "
+      + f"{t3:.5f} s")
+
+###############################################################################
+# Plot selected frequency bands
+# ----------------------------------
+#
+# Plot the class distinctiveness values for each sub_band,
+# along with the highlight of the finally selected frequency band.
+
+subband_fmin = list(np.arange(freq_band[0],
+                              freq_band[1] - sub_band_width + 1.,
+                              sub_band_step))
+subband_fmax = list(np.arange(freq_band[0] + sub_band_width,
+                              freq_band[1] + 1., sub_band_step))
+nb_subband = len(subband_fmin)
+
+x = list(range(0, nb_subband, 1))
+fig = plt.figure(figsize=(10, 5))
+
+freq_start = subband_fmin.index(best_freq[0])
+freq_end = subband_fmax.index(best_freq[1])
+
+plt.subplot(1, 1, 1)
+plt.grid()
+plt.plot(x, all_class_dis, marker='o')
+plt.xticks(list(range(0, 14, 1)),
+           ["[5, 9]", "[7, 11]", "[9, 13]", "[11, 15]", "[13, 17]",
+            "[15, 19]", "[17, 21]", "[19, 23]", "[21, 25]",
+            "[23, 27]", "[25, 29]", "[27, 31]", "[29, 33]", "[31, 35]"])
```

Comment:
I don't like this. This should be constructed from subband_fmin and subband_fmax